### ISSUE TITLE
[iOS] Show username not just display name

### ISSUE THREAD

**Issue by mblarsen** [gitterHQ/gitter#1325]
While I love being able to use gitter "on the go" I've stopped using the app for one simple reason. It is super hard to follow a conversation between people because the mentions in a post is username while the avatar name is the display name. A lot of people does not have a username looking like their display. As a result you cannot find out who is saying what unless you click their profile => github and then trying to guess it that way.

![gitter-username](https://cloud.githubusercontent.com/assets/247048/16510752/612da3da-3f7b-11e6-982d-b939d3173e94.jpg)

Here is the same conversation on the desktop and web app:

![gitter-username-desktop](https://cloud.githubusercontent.com/assets/247048/16510784/b9c5516e-3f7b-11e6-9123-0337b83f2db4.jpg)

**Comment by ferventcoder** [comment]
This would be great!
